javascript/builtins/Intl.json

```diff
@@ -1082,6 +1089,70 @@
               }
             }
           },
+          "formatToParts": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/formatToParts",
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": "63",
+                  "flag": {
+                    "type": "preference",
+                    "name": "harmony-number-format-to-parts"
+                  },
+                  "notes": "Need to set the flag on the commandline via --js-flag"
+                },
+                "chrome_android": {
+                  "version_added": "63",
+                  "flag": {
+                    "type": "preference",
+                    "name": "harmony-number-format-to-parts"
+                  },
+                  "notes": "Need to set the flag on the commandline via --js-flag"
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": null
+                },
+                "firefox_android": {
+                  "version_added": null
+                },
+                "ie": {
+                  "version_added": null
+                },
+                "ie_mobile": {
+                  "version_added": null
+                },
+                "nodejs": {
+                  "version_added": null
+                },
+                "opera": {
+                  "version_added": null
+                },
+                "opera_android": {
+                  "version_added": null
+                },
+                "safari": {
+                  "version_added": null
+                },
+                "safari_ios": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": true,
```

Comment:
You changed Intl.DateTimeFormat.prototype.formatToParts to be longer experimental, I guess Intl.NumberFormat.prototype.formatToParts should then also not be experimental, or am I wrong?

Reply:
The former is a part of Ecma 402 2017, but the latter is not yet as far as I can tell.

Reply:
Okay, I didn't dig too deep in the specs, thanks!